sdks/python/src/opik/evaluation/tasks_scorer.py

```diff
@@ -129,7 +170,13 @@ def run(
     with futures.ThreadPoolExecutor(max_workers=workers) as pool:
         test_case_futures = [
             pool.submit(
-                _process_item, client, item, task, scoring_metrics, project_name
+                _process_item,
```

Comment:
I think this call has grown big enough to start passing arguments as key-value pairs `submit(_process_item, client=client, item=item, ...)`

Reply:
@alexkuzmik Didn't know you could do that, much cleaner I'll update